Apps/WebClient/src/Server/Models/DependentModel.cs

```diff
@@ -29,6 +29,12 @@ public class DependentModel
         [JsonPropertyName("name")]
         public string Name { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets the HhdId of the dependent.
+        /// </summary>
+        [JsonPropertyName("hdId")]
+        public string HdId { get; set; } = string.Empty;
```

Comment:
Vue needs dependent's HdId for the COVID-19 tab (design has been discussed with @sslaws)